cmd/vulcan-exposed-rdp/main.go

```diff
@@ -121,12 +122,26 @@ func isExposedRDP(host, port string, timeout int) []report.Vulnerability {
 
 	if CCPDURegex.Match(b) && b[5] == byte('\xd0') {
 		exposedRDP.Details += fmt.Sprintf("RDP host detected in port: %s", port)
+		exposedRDP.AffectedResource = fmt.Sprintf("%s", port)
+		exposedRDP.ID = computeVulnerabilityID(host, exposedRDP.AffectedResource, port)
```

Comment:
Now there's a new `Vulnerability.Fingerprint` attributed to be used instead of the `ID`. Also we are not adding the target nor the affected resource to calculate the fingerprint now.

Suggested change:
```
v.Fingerprint = helpers.ComputeFingerprint()
```
(Adding the port it's harmless and we can keep it if you want)

Reply:
Okey! Using the `Fingerprint` attribute now, also removed all args in the fingerprint calculation.